View Controllers/SearchViewController.m

```diff
@@ -76,17 +77,55 @@ - (void)initUIStyles {
     [self.navigationController.navigationBar setShadowImage:[UIImage new]];
 }
 
+- (void)initMyLocation {
+    self.locationManager = [CLLocationManager new];
+    self.locationManager.desiredAccuracy = kCLLocationAccuracyBest;
+    self.locationManager.delegate = self;
+    [self.locationManager startUpdatingLocation];
+}
+
 - (void)fetchPlaces {
-    [[APIManager shared] getNextGMSPlacesBatch :^(NSArray<GMSPlace *> *places) {
-        if (!self.places) {
-            self.places = places;
-        }
-        else {
-            self.places = [self.places arrayByAddingObjectsFromArray:places];
-        }
-        self.filteredPlaces = self.places;
-        [self.tableView reloadData];
-        self.isMoreDataLoading = NO;
+    const double kRange = 1.0;
```

Comment:
teeny tiny nit, when we have constants at the method level, we don't usually start with k, we only do it on class level static consts :)